docs/developers/torch_add_a_new_operator.md

```diff
@@ -0,0 +1,121 @@
+# How To Add a New Torch Operator
+
+![Torch-MLIR Architecture](./pics/Torch-MLIR-MHLO.png)
+
+TorchBlade converts PyTorch workloads to MHLO based on [Torch-MLIR](https://github.com/llvm/torch-mlir/).
+Then compile the MHLO modules via BladeDISC compiler.
+
+The BladeDISC Dev Team is cooperating with the community to add Torch-To-Mhlo
+conversion to Torch-MLIR, especially fully dynamic shape features.
+See RFC: https://github.com/llvm/torch-mlir/issues/999.
+We appeal to the community developers interested in joining.
+
+## To the Developers of Torch-MLIR
```

Comment:
maybe the readers of this document are all BladeDISC developer, let's remove this section? Or move the reference docs to the end of this document.